test/runtime/manifests/l7-simple.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
can you rename these policy files to correspond to the test in which they are used ? 
e.g., Policies-l7-simple.json